mino/minogrpc/controller/mod.go

```diff
@@ -1,26 +1,43 @@
 package controller
 
 import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"io"
 	"math"
+	"path/filepath"
 	"time"
 
 	"go.dedis.ch/dela"
 	"go.dedis.ch/dela/cli"
 	"go.dedis.ch/dela/cli/node"
+	"go.dedis.ch/dela/core/store/kv"
+	"go.dedis.ch/dela/crypto/loader"
 	"go.dedis.ch/dela/mino"
 	"go.dedis.ch/dela/mino/minogrpc"
+	"go.dedis.ch/dela/mino/minogrpc/certs"
 	"go.dedis.ch/dela/mino/router/tree"
 	"golang.org/x/xerrors"
 )
 
+const certKeyName = "cert.key"
+
 // Minimal is an initializer with the minimum set of commands.
 //
 // - implements node.Initializer
-type minimal struct{}
+type minimal struct {
```

Comment:
I have a bit of an issue with name.
`minimal` is an adjective, it doesn't quite describe what this is/does.
Can you think of a better name?